lib/finch/pool/strategy.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Finch.Pool.Strategy do
```

Comment:
These new Pool modules should probably be private/@moduledoc false, but i felt the documentation could be useful when implementing additional strategies